serverless/cmd/integrate/main.go

```diff
@@ -128,15 +127,24 @@ func main() {
 	}
 }
 
+func getKeyFile(path string) (key string, err error) {
+	k, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("failed to read key file: %q", err)
+	}
+	key = string(k)
```

Comment:
can inline this too `return string(k), nil`

Reply:
Done